crates/uv/tests/pip_sync.rs

```diff
@@ -65,7 +65,7 @@ fn missing_venv() -> Result<()> {
     ----- stdout -----
 
     ----- stderr -----
-    error: No virtual environment found
+    error: No virtual environment found; run `uv venv` to create an environment, or pass `--system` to install into a non-virtual environment
```

Comment:
Could probably also have test coverage for the case `--system` with no Python installations, `TexrContext::new_with_versions([])` should be get you the proper setup.

Reply:
Done!